src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -1,25 +1,32 @@
 @use 'sass:map';
 @use '../../material/core/style/layout-common';
+@use '../../material/core/style/focus-indicators';
 
 /// Mixin that turns on strong focus indicators.
 ///
 /// @example
 ///   .my-app {
 ///     @include mat-mdc-strong-focus-indicators($config);
 ///   }
+
 @mixin strong-focus-indicators($config: ()) {
+  @include focus-indicators.private-strong-focus-indicators-helper();
```

Comment:
I think that this call can be removed since it's also being called a few lines down.